tests/functional/workloads/app/amq/test_amq_pod_respin.py

```
@@ -23,20 +23,40 @@ def respin_amq_app_pod(kafka_namespace, pod_pattern):
+                    f"expected_count={expected_pod_count}, all_running={pods_running}"
+                )
+                assert pods_running, f"Not all pods recovered after deleting {pod_name}"
+                logger.info(
+                    f"Pod {pod_name} respun successfully, all {expected_pod_count} pods running"
                 )
                 break
         except IndexError as ie:
-            log.error(" pod doesn't exist")
+            logger.error(
+                f"Pod matching pattern '{pod_pattern}' doesn't exist in namespace {kafka_namespace}"
+            )
```

Ruff (0.15.15)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)